codegen/tests/test_modules.rs

```diff
@@ -222,6 +222,73 @@ fn duplicate_fn_rename_test() -> Result<(), Box<EvalAltResult>> {
     Ok(())
 }
 
+mod multiple_fn_rename {
```

Comment:
You covered more than I asked for! I might refactor this test myself later based on my getter-setter work, but this is totally fine.